### PR TITLE
Add migration removing billing data from personal teams

### PR DESCRIPTION
**Pull request by dan-f**
@mattkrick this should address 1/2 of #1452

### CODE REVIEW

File: src/server/database/migrations/20171026160540-removeBillingFromPersonalTeams.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    'periodEnd',
+    'periodStart'
+  ];
+  return r
+    .table('Organization')
+    .filter({tier: 'personal'})
+    .replace((org) =>
+      org
+        .without(...billingFields)
+        .merge({creditCard: {}})
```
Comment (dan-f) [inline src/server/database/migrations/20171026160540-removeBillingFromPersonalTeams.js]: I did this based on what I was seeing in the system already. That said, intuitively it feels like this field shouldn't exist on teams that aren't (or aren't planning to) pay for a pro account yet. Anyways, I'm curious as to your thoughts.

Comment (mattkrick) [reply src/server/database/migrations/20171026160540-removeBillingFromPersonalTeams.js]: Yeah, removing it should be OK, but we should search for `creditCard` to make sure it supports a null whenever we call it.

**Comment by mattkrick** [comment]
We still need to call stripe and tell it to remove these users. In the past, I've put it inside a production conditional in the migration. That's not ideal, but it's nice to know what we did and when. Alternatively, we could just write a script, run it, and then save it in a file somewhere never to be called again. I'm open to any other idea you got!

**Comment by dan-f** [comment]
> We still need to call stripe and tell it to remove these users. In the past, I've put it inside a production conditional in the migration. That's not ideal, but it's nice to know what we did and when. Alternatively, we could just write a script, run it, and then save it in a file somewhere never to be called again. I'm open to any other idea you got!

I'd prefer to avoid the production conditional approach in order to minimize side-effects and prevent stripe errors from taking down our migrations. I was thinking we could collect these one-off scripts somewhere... More on that in slack.

**Comment by mattkrick** [comment]
@dan-f wonderful! would you mind bumping the changelog & pushing to staging & prod?

**Comment by dan-f** [comment]
Not at all!

**Pull request merged**